src/sentry/static/sentry/app/components/avatar/avatarList.jsx

```diff
@@ -19,31 +22,63 @@ export default class AvatarList extends React.Component {
   };
 
   render() {
-    const {users, avatarSize, maxVisibleAvatars} = this.props;
+    const {
+      users,
+      avatarSize,
+      maxVisibleAvatars,
+      tooltipOptions,
+      renderTooltip,
+    } = this.props;
     const visibleUsers = users.slice(0, maxVisibleAvatars);
     const numCollapsedUsers = users.length - visibleUsers.length;
 
     return (
       <Flex direction="row-reverse">
-        {visibleUsers.map(user => {
-          return <StyledAvatar key={user.id} user={user} size={avatarSize} hasTooltip />;
-        })}
         {!!numCollapsedUsers && (
-          <CollapsedUsers size={avatarSize}>{numCollapsedUsers}</CollapsedUsers>
+          <Tooltip title={`${numCollapsedUsers} other users`}>
+            <CollapsedUsers size={avatarSize}>
+              <Plus>+</Plus>
+              {Math.max(numCollapsedUsers, 99)}
```

Comment:
Is this something to worry about? Are there teams with > 99 (+maxVisible) users?

Reply:
I assume there's no limit, so setting a max of 99 could be incorrect. Where does the 99 number come from anyway?

Reply:
Trying to prevent overflow when there's 3 digits. Maybe we can move the plus so that from 1-98 it's `+<number>` and then it becomes `99+`

Reply:
Doesn't the count always show 99 in that case?

Reply:
oops yeah, should be min.

Reply:
Removing the plus would be fine

Reply:
I think you meant Max.min then?

Reply:
In any case, I think we should drop it and just remove the plus, or change the font size or something along those lines